packages/at_secondary_server/lib/src/server/at_security_context_impl.dart

```diff
@@ -1,38 +1,30 @@
 import 'package:at_secondary/src/server/at_secondary_config.dart';
-import 'package:at_server_spec/at_server_spec.dart';
 
-class AtSecurityContextImpl implements AtSecurityContext {
```

Comment:
AtSecurityContext is a useless abstraction, there is no reason to have a dependency on at_server_spec for it